packages/office-ui-fabric-react/src/components/Text/examples/Text.Block.Example.tsx

```diff
@@ -0,0 +1,12 @@
+// @codepen
+import * as React from 'react';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+
+export const TextBlockExample = () => (
+  <div>
```

Comment:
Consider `React.Fragment` in stead of an empty DIV unless it modifies example behavior.